apps/src/feedback.js

```diff
@@ -203,7 +203,8 @@ FeedbackUtils.prototype.displayFeedback = function (options, requiredBlocks,
       this.calculateStageProgress(
         actualBlocks <= idealBlocks,
         hintsUsed,
-        options.response.level_id) :
+        options.response.level_id,
+        Number.isFinite(idealBlocks)) :
```

Comment:
I think babel-polyfill would have translated it for us since `Number.isFinite` is standard ES6, but using the global is shorter anyway.